### DISCUSSION TITLE
Show tag in overview (column)

### DISCUSSION THREAD

**Discussion by UsefulVid** [keepassxreboot/keepassxc#10746] [Q&A]
Is it possible to show tags from an entry in the overview?
![image](https://github.com/keepassxreboot/keepassxc/assets/39967070/63237de2-1cb7-4b08-b816-8b856930abd8)
Would this be worth a feature request?

**Comment by droidmonkey** [comment]
I'm curious on the use case for this, can you elaborate?

**Reply by UsefulVid** [reply]
For example if one person also has entries for a second person in keepass and my idea was to separate the entries via the tag.
Especially for older couples in which only one person is more digital then the other :)

**Reply by droidmonkey** [reply]
I would recommend a group for that situation since it's more obvious. Also clicking a tag in the lower left area will show all entries associated with that tag (can also select more than one). Tags are kind of a "meta group" and not a sorting mechanism.

**Reply by UsefulVid** [reply]
The group is no solution for me. For example both use the same bank or insurance. I use groups for the insurance and the bank.

**Reply by weaverbx-risk** [reply]
I'd like to sort by tags, or ideally be able to add a custom field and add that as a column and sort by that.
